packages/server/lib/browsers/memory/index.ts

```diff
@@ -73,8 +82,13 @@ const measure = (func: (...args) => any, opts: { name?: string, save?: boolean }
   }
 }
 
-export const getJsHeapSizeLimit = measure(async (automation: Automation): Promise<number> => {
-  let heapLimit
+/**
+ * Retrieves the JS heap size limit for the browser.
+ * @param automation the automation client to use
```

Comment:
```suggestion
 * @param automation - the automation client to use
```

Reply:
The formatting is automatically applied (at least in VSCode):
<img width="517" alt="Screen Shot 2023-01-17 at 8 58 45 AM" src="https://user-images.githubusercontent.com/2002044/212947307-aea7240f-7ef8-40ab-9a4b-c2093734b86c.png">

Reply:
what view is this?

Reply:
Hovering over the function

Reply:
I must not have this extension or configured? I was just taking from reading through it. It's a nit really but makes it easier to read IMO

Reply:
updated [6541f7d](https://github.com/cypress-io/cypress/pull/25462/commits/6541f7dfdc7b4fd8e90e99734358c01a2fee4fea)